lib/request.js

```diff
@@ -61,17 +124,29 @@ function transformResult (result) {
 	// Then stringify it.
 	let body = null;
 	if (data instanceof Buffer) {
-		body = data.toString(encoding);
+		body = data.toString(encoding || 'binary');
```

Comment:
Do we need `||` here? It looks like encoding will be always `'binary'` even if it was not found in headers or file content